code/modules/multiz/movement.dm

```diff
@@ -184,6 +184,12 @@
 	if((locate(/obj/structure/disposalpipe/up) in below) || locate(/obj/machinery/atmospherics/pipe/zpipe/up in below))
 		return FALSE
 
+/mob/living/can_fall()
+	var/turf/below = GetBelow(src)
+	if(!below || (locate(/obj/structure/ladder) in GetBelow(src)))
```

Comment:
This needs a logical flipping.

`if (below && (locate(/obj/structure/ladder) in below))`

Otherwise it'll handle a broader spectrum of cases which it really shouldn't.